addons/project/static/src/js/project_name_with_subtask_count_widget.js

```diff
@@ -5,6 +5,14 @@ odoo.define('project.name_with_subtask_count', function (require) {
     const FieldChar = require('web.basic_fields').FieldChar;
 
     const FieldNameWithSubTaskCount = FieldChar.extend({
+        // Remove event handlers on this widget to ensure that the kanban 'global
+        // click' opens the clicked record, even if the click is done on a title of task
+        // This is necessary because of the weird 'global click' logic in
+        // KanbanRecord, which should definitely be cleaned.
+        // Anyway, those handlers are only necessary in Form and List views, so we
+        // can removed them here.
+        events: _.omit(FieldChar.prototype.events, 'click'),
+
```

Comment:
@uso-odoo Thanks a lot for all your researches ! It's a weird corner case.

We do think with @IT-Ideas et @xavierbol that we could more explicitly write :

```suggestion
// As the purpose of this widget is to render the field as a readonly value,
// we don't handle any event on this field.
events: {},
```

What do you think about it ?

Reply:
@tlibioulle , @IT-Ideas ,
Actually, This widget is also used inside the list view. and when we have multi_edit mode on task list then it will break the edit task name if we does not handle any event.
that's why only omitting the click event should be better.

what do you think?

Thanks

Reply:
Hello @mba-odoo ,

Wow, thank you for having pointed that out!
Couldn't we detect the view type somewhere (init, start, click override?) and react accordingly then?

Thanks!

Reply:
Hello @IT-Ideas ,
We may create a two separate widget(one without event and one with event) and register them with view like: `kanban. name_with_subtask_count ` and `list.name_with_subtask_count ` so it will call individual widget according to view type.

Thanks

Reply:
Hi @mba-odoo , ( @IT-Ideas )

Thanks for your vigilance !

And if we write something like this : (omit click only on kanban view). Just to have only one widget and not two ?

```suggestion
        init: function () {
            this._super.apply(this, arguments);
            if (this.viewType === 'kanban') {
                const { click, ...events} = this.events;
                this.events = events;
            }
        },
```

Reply:
@tlibioulle Done the changes. Thanks

Reply:
LGTM :) Thank you !